Description: Correct pod NAME issues in LedgerSMB/DBObject/Admin.pm
 Has the module name but does not have the dash plus short description in the
 NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472546&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/DBObject/Admin.pm
+++ b/LedgerSMB/DBObject/Admin.pm
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-LedgerSMB::DBObject::Admin 
+LedgerSMB::DBObject::Admin - User/group management for LedgerSMB
 
 =head1 SYNOPSIS
 
